src/Uno.UI/UI/Xaml/Media/ImageBrush.wasm.cs

```diff
@@ -1,4 +1,8 @@
-﻿using Windows.Foundation;
+﻿using System;
+using Windows.Foundation;
+using Windows.UI.Xaml.Wasm;
+using Uno.Disposables;
+using NotImplementedException = System.NotImplementedException;
```

Comment:
I gotta know... what other NotImplementedExceptions are there?

Reply:
That's strange... specially because it's not used at all in the file.